packages/account/src/Sections/Profile/PersonalDetails/verify-button.tsx

```diff
@@ -8,94 +8,105 @@ import { useTranslations, Localize } from '@deriv-com/translations';
-            const timeout = setTimeout(() => {
-                if (ref.current) {
-                    ref.current.style.scrollMarginTop = isDesktop ? '120px' : '80px';
-                    ref.current.scrollIntoView({ behavior: 'smooth' });
-                }
-            }, 0);
+        useEffect(() => {
+            if (is_scroll_to_verify_button) {
+                // To make scrolling work on mobile we need to add a delay.
+                const timeout = setTimeout(() => {
+                    if (ref.current) {
+                        ref.current.style.scrollMarginTop = isDesktop ? '120px' : '80px';
+                        ref.current.scrollIntoView({ behavior: 'smooth' });
+                    }
+                }, 0);
 
-            return () => {
-                clearTimeout(timeout);
-                setIsScrollToVerifyButton(false);
-            };
-        }
-    }, [is_scroll_to_verify_button, setIsScrollToVerifyButton, isDesktop]);
+                return () => {
+                    clearTimeout(timeout);
+                    setIsScrollToVerifyButton(false);
+                };
+            }
+        }, [is_scroll_to_verify_button, setIsScrollToVerifyButton, isDesktop]);
 
-    useEffect(() => {
-        if (WS.isSuccess) {
-            history.push(routes.phone_verification);
-        }
-    }, [WS.isSuccess, history]);
+        useEffect(() => {
+            if (WS.isSuccess) {
+                history.push(routes.phone_verification);
+            } else if (error) {
+                //@ts-expect-error error message has issue with typing
+                setStatus({ msg: error.message, code: error.code });
+            }
+        }, [WS.isSuccess, history, error]);
 
-    const redirectToPhoneVerification = (e: React.MouseEvent<HTMLElement>) => {
-        e.preventDefault();
-        setVerificationCode('', 'phone_number_verification');
-        setShouldShowPhoneNumberOTP(false);
-        sendPhoneNumberVerifyEmail();
-    };
+        const redirectToPhoneVerification = (e: React.MouseEvent<HTMLElement>) => {
+            e.preventDefault();
+            setVerificationCode('', 'phone_number_verification');
+            setShouldShowPhoneNumberOTP(false);
+            sendPhoneNumberVerifyEmail();
+        };
 
-    const verifyTimer = () => {
-        let resendCodeTimer = '';
-        if (next_email_otp_request_timer) {
-            next_email_otp_request_timer < 60
-                ? (resendCodeTimer = `${localize(' in ')}${next_email_otp_request_timer}s`)
-                : (resendCodeTimer = `${localize(' in ')}${Math.round(next_email_otp_request_timer / 60)}m`);
-        } else {
-            resendCodeTimer = '';
-        }
+        const verifyTimer = () => {
+            let resendCodeTimer = '';
+            if (next_email_otp_request_timer) {
+                next_email_otp_request_timer < 60
+                    ? (resendCodeTimer = `${localize(' in ')}${next_email_otp_request_timer}s`)
```

Comment:
cab we double check if localize is working properly here?

Reply:
Yeappp localize is working properly here 🙏